src/components/Links/ButtonLink.tsx

```diff
@@ -2,16 +2,32 @@ import classNames from 'classnames';
 import { Link } from 'gatsby';
 import React from 'react';
 
-type Props = { to: string; className?: string; newWindow?: boolean };
+type ButtonSizes = 'sm' | 'md' | 'lg';
+
+type Props = {
+  to: string;
+  className?: string;
+  newWindow?: boolean;
+  size?: ButtonSizes;
+  outline?: boolean;
+};
 
 // TODO: Refactor all those link components to share more logic
 export const ButtonLink: React.FC<Props> = ({
   to,
   className = '',
   newWindow = false,
+  outline = false,
+  size = 'md',
   children,
 }) => {
-  const styles = classNames(className, 'btn-brand');
+  const styles = classNames(
+    className,
+    { 'btn-brand-outline': outline },
+    { 'w-full px-8 py-3 md:text-lg': size === 'lg' },
+    { 'px-3 py-1 text-sm font-medium': size === 'sm' },
+    'btn-brand'
```

Comment:
I think that would be the perfect situation to move the styles of btn-brand back here. We only use it here ([Search](https://github.com/FixMyBerlin/fixmy.rsv/search?q=btn-brand)), so there is no reason to extract it in a "css"-component. The rule of thumb for Tailwind CSS is: Keep it inside the component until you re-use it somewhere or extensively.